docs/install/Knative-with-any-k8s.md

```diff
@@ -84,7 +84,7 @@ your Knative installation, see
    kubectl apply --filename https://github.com/knative/serving/releases/download/v0.6.0/serving.yaml --selector networking.knative.dev/certificate-provider!=cert-manager \
    --filename https://github.com/knative/build/releases/download/v0.6.0/build.yaml \
    --filename https://github.com/knative/eventing/releases/download/v0.6.0/release.yaml \
-   --filename https://github.com/knative/eventing-sources/releases/download/v0.6.0/eventing-sources.yaml \
+   --filename https://github.com/knative/eventing-contrib/releases/download/v0.6.0/eventing-sources.yaml \
```

Comment:
@n3wscott  -- just to confirm, when we rename the repo to contrib, are we still going to release an "eventing-sources.yaml" file, or will that be renamed as well? We can open an issue about updating this later in the latter is true, just to make sure we don't miss it.